src/config.ts

```diff
@@ -373,14 +375,12 @@ export const ASSET_SWAPPER_MARKET_ORDERS_V0_OPTS: Partial<SwapQuoteRequestOpts>
     shouldGenerateQuoteReport: false,
 };
 
-export const BASE_GAS_COST_V1 = new BigNumber(1.3e5);
-
 // We are keeping cost in gross gas for just the portion of the fill
 // overhead is then later added ontop of the conservative estimate
 export const GAS_SCHEDULE_V1: FeeSchedule = {
     [ERC20BridgeSource.Native]: () => 1.5e5,
-    [ERC20BridgeSource.Uniswap]: () => 1.1e5,
-    [ERC20BridgeSource.LiquidityProvider]: () => 1.3e5,
+    [ERC20BridgeSource.Uniswap]: () => 9e5,
+    [ERC20BridgeSource.LiquidityProvider]: () => 1.4e5,
```

Comment:
Bumping this up a bit because now PLP is only operating on USDC-DAI and USDC-USDT, both of which are expensive asset pairs